### DISCUSSION TITLE
Jax Scipy RBFInterpolator?

### DISCUSSION THREAD

**Discussion by FluffyCodeMonster** [jax-ml/jax#22503] [Ideas]
I love the fact that Jax has implemented some of the functionality of the scipy.interpolate module, specifically the RegularGridInterpolator. However, currently only the linear and nearest neighbour interpolations are supported, which is a problem when the interpolation needs to be smooth (e.g. for running gradient-based optimisers). How difficult would it be to 'jaxify' the scipy.interpolate.RBFInterpolator class and related code, for smooth interpolations? I've made a start at this myself, but it's a little beyond me at the moment. I feel like this could be an asset to the Jax framework.

**Comment by jakevdp** [comment]
We've made the decision that `scipy.interpolate` is out-of-scope for the JAX core package. That doesn't mean it's not useful to some people, but it means we'd encourage the community to develop those implementations elsewhere.

See [JEP 18137](https://jax.readthedocs.io/en/latest/jep/18137-numpy-scipy-scope.html#scipy-interpolate) for a discussion.
